impeller/entity/contents/filters/filter_contents.cc

```diff
@@ -55,23 +56,38 @@ std::shared_ptr<FilterContents> FilterContents::MakeGaussianBlur(
     Sigma sigma_y,
     BlurStyle blur_style,
     Entity::TileMode tile_mode) {
-  std::shared_ptr<FilterContents> x_blur = MakeDirectionalGaussianBlur(
-      /*input=*/input,
-      /*sigma=*/sigma_x,
-      /*direction=*/Point(1, 0),
-      /*blur_style=*/BlurStyle::kNormal,
-      /*tile_mode=*/tile_mode,
-      /*is_second_pass=*/false,
-      /*secondary_sigma=*/{});
-  std::shared_ptr<FilterContents> y_blur = MakeDirectionalGaussianBlur(
-      /*input=*/FilterInput::Make(x_blur),
-      /*sigma=*/sigma_y,
-      /*direction=*/Point(0, 1),
-      /*blur_style=*/blur_style,
-      /*tile_mode=*/tile_mode,
-      /*is_second_pass=*/true,
-      /*secondary_sigma=*/sigma_x);
-  return y_blur;
+  constexpr bool use_new_filter =
+#ifdef IMPELLER_ENABLE_NEW_GAUSSIAN_FILTER
+      true;
+#else
+      false;
+#endif
+
+  // TODO(https://github.com/flutter/flutter/issues/131580): Remove once the new
```

Comment:
Can you document (here or in the issue) which cases the blur doesn't handle now?

Reply:
I have them listed in the issue above.  If we land this early I can copy them over to the issue.